### PR TITLE
feat(aged debtor) change currency

### LINKED ISSUE #3210: Allow Aged Debtors to show values in a non-enterprise currency
The Aged Debtor Report only supports the enterprise currency at this point.  It would be helpful to show the values in a non-enterprise currency.  It should function similarly to the Account Statement Report currency select, among others.

### PR DESCRIPTION
**Pull request by jeremielodi**
closes  #3210

### CODE REVIEW

File: server/models/functions.sql [hunk 1]
```diff
@@ -56,6 +57,23 @@ BEGIN
...
+  projectId INT,
+  currencyId INT,
+  date TIMESTAMP
+)
+RETURNS DECIMAL(19, 8) DETERMINISTIC
+BEGIN
+  DECLARE _enterprise_id smallint(5) unsigned;
+  SELECT enterprise_id INTO _enterprise_id FROM project WHERE id = projectId;
+
+  RETURN (SELECT GetExchangeRate(_enterprise_id, currencyId, date ));
```
Comment (jniles) [inline server/models/functions.sql]: Just FYI, you could simplify this by putting it in one statement:
```sql
RETURN (SELECT GetExchangeRate(p.enterprise_id, currencyId, date) FROM project p WHERE p.id = projectId);
```

File: server/controllers/finance/reports/debtors/index.js [hunk 2]
```diff
@@ -85,19 +86,35 @@ async function queryContext(params = {}) {
   // format the dates for MySQL escape
   const dates = _.fill(Array(5), params.date);
   const data = {};
+  const currencyId = db.escape(params.currency_id);
```
Comment (jniles) [inline server/controllers/finance/reports/debtors/index.js:89]: Nice escaping! 👍

File: server/controllers/finance/reports/debtors/index.1.js [hunk 3]
```diff
@@ -0,0 +1,171 @@
+/**
```
Comment (jniles) [inline server/controllers/finance/reports/debtors/index.1.js]: Why does this file exist..?

**Review by jniles — APPROVED** [review]
This looks really good!  It looks like you accidentally pushed an extra file - could you remove that?  Then it can be merged!

File: server/controllers/finance/reports/debtors/index.1.js (hunk 3 above)
Comment (jeremielodi) [reply server/controllers/finance/reports/debtors/index.1.js]: it should be removed

**Review by jniles — APPROVED** [review]
Excellent work!

bors r+

**Pull request merged**